Source/WebCore/platform/network/cocoa/CookieCocoa.mm

```diff
@@ -147,6 +149,9 @@ static NSHTTPCookieStringPolicy _Nullable nsSameSitePolicy(Cookie::SameSitePolic
     if (!path.isNull())
         [properties setObject:(NSString *)path forKey:NSHTTPCookiePath];
 
+    if (!partitionKey.isNull())
+        [properties setObject:(NSString *)partitionKey forKey:@"StoragePartition"];
```

Comment:
Is there no "NSHTTPCookiePartition" or similarly named `NSHTTPCookiePropertyKey`? If not, so we know if this is going to be added?

Reply:
Unfortunately no, it's not currently exposed, but I'll work on doing that.